### PR TITLE
feat(playbutton): change play button text based on userData

### PR DESCRIPTION
**Pull request by Camc314**
Changes play button text depending on user watch status

 - Play Episode X
 - Resume Episode X
 - Play Movie
 - Resume Movie

Falls back to play if no user data or if item type is not series/movie

### CODE REVIEW

File: pages/item/_itemId/index.vue [hunk 1]
```diff
@@ -120,6 +130,32 @@ export default Vue.extend({
       }
       return response.join(' • ');
     },
+    renderPlayButton(): string {
+      console.log(this);
```
Comment (KristupasSavickas) [inline pages/item/_itemId/index.vue:134]: 😱

Comment (Camc314) [reply pages/item/_itemId/index.vue:134]: Oops

Comment (Camc314) [reply pages/item/_itemId/index.vue:134]: Bad habits

File: pages/item/_itemId/index.vue [hunk 2]
```diff
@@ -120,6 +130,31 @@ export default Vue.extend({
       }
       return response.join(' • ');
     },
+    renderPlayButton(): string {
```
Comment (MrTimscampi) [inline pages/item/_itemId/index.vue:133]: Should be a computed property like `playButtonText`.

**Review by MrTimscampi — CHANGES_REQUESTED** [review]
We should probably start to break this into more sub-components.

I could see one for the play button (potentially combining it with the circle buttons for the cards, to have one component used everywhere) and one for the subtitle line with date, runtime and so on (The equivalent of jf-web's mediaInfo module).

File: pages/item/_itemId/index.vue (hunk 2 above)
Comment (MrTimscampi) [reply pages/item/_itemId/index.vue:133]: @Camc314 Any news on this?

**Comment by MrTimscampi** [comment]
Closing this, since #184 only has "Play" in the buttons

**Pull request closed without merging**